app/controllers/accounts/connected_accounts_controller.rb

```diff
@@ -16,7 +18,15 @@ def show
         sp_name: decorated_sp_session.sp_name,
         user: current_user,
         locked_for_session: pii_locked_for_session?(current_user),
+        all_emails_requested: all_emails_requested?,
       )
     end
+
+    def all_emails_requested?
+      if decorated_sp_session.requested_attributes.present?
+        decorated_sp_session.requested_attributes.map.include?('all_emails') ||
+          !decorated_sp_session.requested_attributes.map.include?('email')
```

Comment:
This `map` isn't doing anything.

```suggestion
        decorated_sp_session.requested_attributes.include?('all_emails') ||
          !decorated_sp_session.requested_attributes.include?('email')
```